pytest_fixtures/core/contenthosts.py

```diff
@@ -98,15 +98,14 @@ def content_hosts(request):
 
 
 @pytest.fixture(scope='module')
-def registered_hosts(organization_ak_setup, content_hosts, module_target_sat):
+def registered_hosts(content_hosts, target_sat, module_org):
```

Comment:
You need to use `module_target_sat` here. Expectation is that our github action running --setup-plan would have caught this.

Rule of thumb here - this is a fixture, so the `target_sat` fixture scope needs to match this local scope.

```
12:57:35  _ ERROR at setup of TestRemoteExecution.test_positive_run_default_job_template_multiple_hosts_by_ip _
12:57:35  ScopeMismatch: You tried to access the function scoped fixture target_sat with a module scoped request object, involved factories:
12:57:35  pytest_fixtures/core/contenthosts.py:100:  def registered_hosts(content_hosts, target_sat, module_org)
12:57:35  pytest_fixtures/core/broker.py:39:  def target_sat(request, _default_sat, satellite_factory)
```